spec/models/piece_spec.rb

```diff
@@ -15,7 +15,7 @@
         password: 'foobar',
         password_confirmation: 'foobar'
       )
-      @game = @user.games.create(white_player_id: @user)
+      @game = @user.games.create(white_player_id: @user.id)
       FROM_ROW = 3
```

Comment:
These constants should be moved out of the `"#move_to!` describe block, as they are being used by all the describe blocks in this file.